checkers/testdata/octalLiteral/negative_tests.go

```diff
@@ -66,7 +65,7 @@ func NoWarningsFs() {
 	_ = fs.FileMode(0o555)
 }
 
-func NoWarningsOs() {
+func NoWarningsOsOpenFile() {
```

Comment:
Alternatively, I could move the `os.OpenFile()` and `os.WriteFile()` calls into a single `NoWarningsOs()`. Flipped a coin and went with the more explicit choice...